### PR TITLE
BUGFIX: Remove migrations

### PR DESCRIPTION
**Pull request by bwaidelich**
Removes doctrine and ContentRepository migrations
and applied core migrations

Related: #4199

### CODE REVIEW

File: Neos.ContentRepositoryRegistry/composer.json [hunk 1]
```diff
@@ -20,87 +20,5 @@
...
-            "Neos.Neos-20161219122512",
-            "Neos.Fusion-20161219130100",
-            "Neos.Neos-20161220163741",
-            "Neos.Neos-20170115114620",
-            "Neos.Flow-20170125103800",
-            "Neos.Seo-20170127154600",
-            "Neos.Flow-20170127183102",
-            "Neos.Fusion-20180211175500",
-            "Neos.Fusion-20180211184832"
-        ]
```
Comment (bwaidelich) [inline Neos.ContentRepositoryRegistry/composer.json:104]: Not related to the issue, but I don't think that we need to keep track of applied core migrations in core packages. It bloats the composer file without real benefits

**Comment by bwaidelich** [comment]
This certainly needs more love (see failing pipeline)
But I think that most failing tests can be removed anyways

**Comment by gradinarufelix** [comment]
@bwaidelich I opened a new PR #4240 because I could not push to this one. Either cherry-pick my changes or abandon this one.

**Comment by bwaidelich** [comment]
@gradinarufelix Thanks, closing this one in favor of #4240

**Pull request closed without merging**